symengine/singleton.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import print_function, division
+from symengine import sympify
```

Comment:
Could you suggest a way of tackling this? I tried a number of ways, but couldn't resolve the issue.